Changes

```diff
@@ -12,6 +12,15 @@ Language features:
 Tools:
 ======
 
+- GPR#514: Added several flags:
```

Comment:
"compiler flags" would be more explicit. You may want to mention that we should always have both the `-foo` and `-no-foo` options, so that people can always explicitly express their intent and be robust if the default setting changes.